test/system/socialworker_test.rb

```diff
@@ -31,8 +31,7 @@ def setup
     @socialworker.clients.each do |client|
       assert page.has_text? client.contact.first_name
       assert page.has_text? client.contact.last_name
-      assert page.has_link? href: client_path(client.id)
-      assert page.has_link? href: edit_client_path(client.id)
+      assert find("a[href='#{client_path(client.id)}']")
```

Comment:
Does `page.has_link? href: client_path(client.id)` not work here?

Reply:
That I will try. Thx